roles/iptables-config/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+# - name: Save firewall-rules
+#   ansible.builtin.shell:
+#     cmd: "/usr/sbin/netfilter-persistent save"
+#     # creates: '/etc/iptables/rules.v4'
+
+- name: Reload firewall-rules
+  ansible.builtin.systemd:
+    name: iptables
```

Comment:
shouldnt here reload nftables if debian12

Reply:
Handler is called only with CentOS7